packages/@aws-cdk/aws-ec2/package.json

```diff
@@ -73,6 +73,7 @@
   },
   "dependencies": {
     "@aws-cdk/aws-cloudwatch": "0.0.0",
+    "@aws-cdk/aws-kms": "0.0.0",
```

Comment:
This should actually be a `devDependency` in the current implementation. If we apply this [comment](https://github.com/aws/aws-cdk/pull/8172#discussion_r440152798) though, than its ok :)